src/layouts/ReviewRequest/components/PublishedModal/PublishedModal.tsx

```diff
@@ -48,15 +69,17 @@ export const PublishedModal = (
 
         <ModalFooter>
           <Button
-            variant="clear"
+            variant="link"
```

Comment:
Can we change this to use the ButtonLink component?

Reply:
unfortunately, no because the hover styling is hard-coded and leads to the bg obscuring the text on hover.